templates/deployment.yaml

```diff
@@ -54,6 +54,11 @@ spec:
           - /etc/docker/registry/config.yml
           ports:
             - containerPort: 5000
+{{- if .Values.configData.http.debug.prometheus.enabled }}
+            - containerPort: {{ (split ":" .Values.configData.http.debug.addr)._1 }}
```

Comment:
Is the debug addr guaranteed to be present? On mobile, so I can't see the whole diff easily to determine whether there's an existing guard clause around this, but since it's referencing something that wasn't being referenced before, I wanted to double check.